pkgs/top-level/python-packages.nix

```diff
@@ -22378,6 +22378,29 @@ in {
     };
   };
 
+  pyslurm = buildPythonPackage rec {
```

Comment:
Can you add only a reference to this file and move the actual expression to a dedicated file `python-modules` as describe in header of this file?